content/zh-cn/blog/_posts/2016-08-00-Sig-Apps-Running-Apps-In-Kubernetes.md

```diff
@@ -0,0 +1,91 @@
+
+SIG Apps is a place where companies and individuals can:-->  
+
+_编者注：这篇文章由 Kubernetes SIG-Apps 团队撰写，分享他们如何关注开发者和开发运维在 Kubernetes 中运行应用程序的经验。_
+
+Kubernetes 是容器化应用程序的出色管理者。因此，[众多](https://kubernetes.io/blog/2016/02/sharethis-kubernetes-in-production)
+[公司](https://blog.box.com/blog/kubernetes-box-microservices-maximum-velocity/)
+[已经](http://techblog.yahoo.co.jp/infrastructure/os_n_k8s/)
+[开始](http://www.nextplatform.com/2015/11/12/inside-ebays-shift-to-kubernetes-and-containers-atop-openstack/) 在 Kubernetes 中运行应用程序。
+
+Kubernetes 特殊兴趣小组 ([SIGs](https://github.com/kubernetes/community/blob/master/README.md#special-interest-groups-sig))
+自 1.0 版本开始就一直致力于支持开发者和运营商社区。围绕网络、存储、扩展和其他运营领域组织的人员。
+
+随着 Kubernetes 的兴起，对工具、最佳实践以及围绕构建和运营云原生应用程序的讨论的需求也随之增加。为了满足这一需求，
+Kubernetes [SIG Apps](https://github.com/kubernetes/community/tree/master/sig-apps) 应运而生。
+
+SIG Apps 为公司和个人提供以下能力：
+
+<!--- see and share demos of the tools being built to enable app operators
+- learn about and discuss needs of app operators
+- organize around efforts to improve the experience-->
+
+- 查看和分享正在构建的工具演示，以支持应用程序操作员
+- 了解和讨论应用运营商的需求
+- 组织各方努力改善体验
+
+<!--Since the inception of SIG Apps we’ve had demos of projects like [KubeFuse](https://github.com/opencredo/kubefuse), [KPM](https://github.com/kubespray/kpm), and [StackSmith](https://stacksmith.bitnami.com/). We’ve also executed on a survey of those operating apps in Kubernetes.  
+
+From the survey results we’ve learned a number of things including:-->  
+
+自从 SIG Apps 成立以来，我们已经进行了项目演示，例如 [KubeFuse](https://github.com/opencredo/kubefuse) 、
+[KPM](https://github.com/kubespray/kpm) ，和 [StackSmith](https://stacksmith.bitnami.com/) 。 
+我们还对 Kubernetes 中的那些运行应用程序进行了调查。
+
+从调查结果中，我们学到了很多东西，包括：
+
+<!-- - That 81% of respondents want some form of autoscaling
+- To store secret information 47% of respondents use built-in secrets. At reset these are not currently encrypted. (If you want to help add encryption there is an [issue](https://github.com/kubernetes/kubernetes/issues/10439) for that.)&nbsp;
+- The most responded questions had to do with 3rd party tools and debugging
+- For 3rd party tools to manage applications there were no clear winners. There are a wide variety of practices
+- An overall complaint about a lack of useful documentation. (Help contribute to the docs [here](https://github.com/kubernetes/kubernetes.github.io).)
+- There’s a lot of data. Many of the responses were optional so we were surprised that 935 of all questions across all candidates were filled in. If you want to look at the data yourself it’s [available online](https://docs.google.com/spreadsheets/d/15SUL7QTpR4Flrp5eJ5TR8A5ZAFwbchfX2QL4MEoJFQ8/edit?usp=sharing).-->
+
+- 81% 的受访者希望采用某种形式的自动扩缩
+- 为了存储秘密信息，47% 的受访者使用内置 Secret 。目前这些资料在重启时并未加密。
+（如果你需要关于加密的帮助，请参见[问题](https://github.com/kubernetes/kubernetes/issues/10439) 。)&nbsp;
```

Comment:
I've proposed to change this sentence in [previous PR comment](https://github.com/kubernetes/website/pull/33587#discussion_r892277998).

Use `（quote）` instead of `（quote)`.
Remove unused `&nbsp;`.
```suggestion
（如果你需要关于加密的帮助，请参见[问题](https://github.com/kubernetes/kubernetes/issues/10439)。）
```